recipe card/recipe card.css

```diff
@@ -13,28 +13,25 @@ body{
     background: linear-gradient(rgb(25, 24, 24, 0.5),rgb(59, 51, 51)),url('background.jpg');
     background-size: cover;
     background-repeat: no-repeat;
-    height: 100vh;;
+    height: 760px;

```

Comment:
### This is what caused the problem
I missed this in last time review. Probably that what caused the rendering problem

_Suggestion_: You can add ` min-height: 100vh;` this will solve the problem without resetting body height.

Setting **body height** to specific value could affect the view negatively